cal/static/jsx/query.jsx

```diff
@@ -58,10 +58,27 @@ var RFiltering = React.createClass({
     }
 })
 
+var RDownload = React.createClass({
+    onClick: function() {
+        var uids = this.props.userList.map(function (user) {
+            return user.id
+        });
+        url = "/isc/?" + $.param({event_ids: uids});
+        // kind of a hack, don't know how else to do it
+        window.location.href = url;
```

Comment:
Check this out.  Are users going to spend any amount of time on the page we're redirecting to?  If that's not a problem, then this way is the accepted way.

http://stackoverflow.com/questions/200337/whats-the-best-way-to-automatically-redirect-someone-to-another-webpage